docs/modeling-your-data/modeling-your-data-with-dbt/dbt-custom-models/index.md

```diff
@@ -31,10 +31,12 @@ The `_this_run` and derived tables are considered part of the 'public' class of
 
 ## What denotes a custom module?
 
-**Does:**  
+**Does:**
+
 In short, anything that plugs into the incremental framework provided by this package. Generally speaking any models you create that reference any of the `_this_run` tables from the standard modules are leveraging this framework and therefore need to be tagged with `snowplow_web/mobile_incremental` (see the [tagging](#tagging-custom-models) section). Such models will typically be materialized as incremental, although for more complex custom modules there may be a series of staging models that ultimately produce a derived incremental model. In this case, all staging models also need to be tagged correctly.
 
-**Doesn't:**  
+**Doesn't:**
+
 Models that only reference a Snowplow web derived table as their input, rather than a `_this_run` table. Since these derived tables are materialized as incremental they contain all historic events. Any models you build that reference these tables can therefore by written in a drop and recompute manner i.e. materialized as a table. This means they do not leverage the incremental framework of this package and therefore **should not be tagged.**
```

Comment:
```suggestion
On the other hand, **what does not count as a custom module are** models that only reference a Snowplow web derived table as their input, rather than a `_this_run` table. Since these derived tables are materialized as incremental they contain all historic events. Any models you build that reference these tables can therefore by written in a drop and recompute manner i.e. materialized as a table. This means they do not leverage the incremental framework of this package and therefore **should not be tagged.**
```